crates/turborepo-auth/src/lib.rs

```diff
@@ -23,8 +23,117 @@ pub enum Token {
 impl Token {
     pub fn into_inner(self) -> String {
         match self {
-            Self::Existing(s) => s,
-            Self::New(s) => s,
+            Self::Existing(token) | Self::New(token) => token.as_str(),
+        }
+    }
+}
+
+fn current_unix_time() -> u128 {
+    use std::time::{SystemTime, UNIX_EPOCH};
+    SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .expect("Time went backwards")
+        .as_millis()
+}
+
+// As of the time of writing, this should always be true, since a token that
+// isn't active returns an error when fetching metadata for the token.
+fn is_token_active(metadata: &ResponseTokenMetadata, current_time: u128) -> bool {
```

Comment:
This feels out of scope, is this meant to be in this PR?

Reply:
whoops, you're right this was a popped stash that snuck in